portal-api/src/v1/gef/routes.js

```diff
@@ -111,4 +115,15 @@ router
   .route('/address/:postcode') // Geospatial Addresses
   .get(validateUserHasAtLeastOneAllowedRole({ allowedRoles: [MAKER, READ_ONLY, ADMIN] }), externalApi.getAddressesByPostcode);
 
+router
+  .route('/facilities/:facilityId/amendments/:amendmentId')
+  .get(
+    validateUserHasAtLeastOneAllowedRole({ allowedRoles: [MAKER] }),
+    mongoIdValidation('facilityId'),
+    mongoIdValidation('amendmentId'),
+    handleExpressValidatorResult,
+    validatePortalFacilityAmendmentsEnabled,
```

Comment:
maybe put this at the top of the chain? I think thats the most helpful response to recieve in this situation?

Reply:
also FWIW could use `.all` since we'll have a `patch` on this url too but might as well leave til later

Reply:
Good point, have moved to the top! And changed to a `.all` 👍